monitoring/src/main/scala/akkaviz/persistence/PersistenceSources.scala

```diff
@@ -0,0 +1,33 @@
+package akkaviz.persistence
+
+import akka.stream.scaladsl.Source
+import io.getquill._
+import io.getquill.naming.SnakeCase
+import org.reactivestreams.Publisher
+import monifu.concurrent.Implicits.globalScheduler
+
+import scala.util.control.NonFatal
+
+object PersistenceSources {
+
+  private[this] lazy val db = source(new CassandraStreamSourceConfig[SnakeCase]("akkaviz.cassandra"))
+
+  def of(ref: String): Source[ReceivedRecord, _] = {
+    try {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://www.codacy.com/app/lustefaniak/akka-viz/file/2256576116/issues/source?bid=3121282&fileBranchId=3181785#l16)